pulsar-broker/src/test/java/org/apache/pulsar/broker/namespace/OwnerShipCacheForCurrentServerTest.java

```diff
@@ -63,4 +68,21 @@ public void testOwnershipForCurrentServer() throws Exception {
                     namespaceServices[i].getOwnerAsync(bundle).get().get().getNativeUrl());
         }
     }
+
+    @Test(timeOut = 30000)
+    public void testCreateTopicWithNotTopicNsOwnedBroker() {
+        String topicName = "persistent://" + TENANT + "/" + NAMESPACE + "/" + "testCreateTopic";
+
+        for (PulsarService pulsarService : this.getPulsarServiceList()) {
```

Comment:
please verify that we found at least one broker

Reply:
Done.